hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+Adds ``numbers.Real``, ``numbers.Rational``, ``typing.Iterable`` , ``typing.Sequence`` , ``typing.Collection`` , ``typing.Sized`` to ``_global_type_lookup``
```

Comment:
The syntax for Sphinx cross-references can be tricky, so I'll just suggest the full message:

```
RELEASE_TYPE: patch

The abstract number classes :class:`~python:numbers.Number`,
:class:`~python:numbers.Complex`, :class:`~python:numbers.Real`,
:class:`~python:numbers.Rational`, and :class:`~python:numbers.Integral`
are now supported by the :func:`~hypothesis.strategies.from_type` 
strategy.  Previously, you would have to use 
:func:`~hypothesis.strategies.register_type_strategy` before they
could be resolved (:issue:`1636`)

Thanks to Superstormer for this patch!
```